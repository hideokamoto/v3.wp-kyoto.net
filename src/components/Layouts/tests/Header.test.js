/* eslint-disable react/jsx-filename-extension */
import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import AppHeader from '../Header'
configure({ adapter: new Adapter() })

describe('AppHeader', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<AppHeader />)
    expect(wrapper).toMatchSnapshot()
  })
})
