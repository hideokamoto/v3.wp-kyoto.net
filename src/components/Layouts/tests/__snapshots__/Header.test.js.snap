// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AppHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Container>
          <Grid
            columns={2}
            relaxed={true}
            verticalAlign="middle"
          >
            <GridRow
              stretched={true}
            >
              <GridColumn>
                <Header
                  as="h1"
                >
                  <HeaderContent>
                    <Link
                      replace={false}
                      to="/"
                    >
                      WP-Kyoto
                    </Link>
                  </HeaderContent>
                  <HeaderSubheader>
                    WordPressとかAWSのブログ
                  </HeaderSubheader>
                </Header>
              </GridColumn>
              <GridColumn>
                <GlobalNav />
              </GridColumn>
            </GridRow>
          </Grid>
        </Container>,
        <Divider />,
      ],
      "className": "App-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Grid
            columns={2}
            relaxed={true}
            verticalAlign="middle"
          >
            <GridRow
              stretched={true}
            >
              <GridColumn>
                <Header
                  as="h1"
                >
                  <HeaderContent>
                    <Link
                      replace={false}
                      to="/"
                    >
                      WP-Kyoto
                    </Link>
                  </HeaderContent>
                  <HeaderSubheader>
                    WordPressとかAWSのブログ
                  </HeaderSubheader>
                </Header>
              </GridColumn>
              <GridColumn>
                <GlobalNav />
              </GridColumn>
            </GridRow>
          </Grid>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <GridRow
              stretched={true}
            >
              <GridColumn>
                <Header
                  as="h1"
                >
                  <HeaderContent>
                    <Link
                      replace={false}
                      to="/"
                    >
                      WP-Kyoto
                    </Link>
                  </HeaderContent>
                  <HeaderSubheader>
                    WordPressとかAWSのブログ
                  </HeaderSubheader>
                </Header>
              </GridColumn>
              <GridColumn>
                <GlobalNav />
              </GridColumn>
            </GridRow>,
            "columns": 2,
            "relaxed": true,
            "verticalAlign": "middle",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <GridColumn>
                  <Header
                    as="h1"
                  >
                    <HeaderContent>
                      <Link
                        replace={false}
                        to="/"
                      >
                        WP-Kyoto
                      </Link>
                    </HeaderContent>
                    <HeaderSubheader>
                      WordPressとかAWSのブログ
                    </HeaderSubheader>
                  </Header>
                </GridColumn>,
                <GridColumn>
                  <GlobalNav />
                </GridColumn>,
              ],
              "stretched": true,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <Header
                    as="h1"
                  >
                    <HeaderContent>
                      <Link
                        replace={false}
                        to="/"
                      >
                        WP-Kyoto
                      </Link>
                    </HeaderContent>
                    <HeaderSubheader>
                      WordPressとかAWSのブログ
                    </HeaderSubheader>
                  </Header>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "as": "h1",
                    "children": Array [
                      <HeaderContent>
                        <Link
                          replace={false}
                          to="/"
                        >
                          WP-Kyoto
                        </Link>
                      </HeaderContent>,
                      <HeaderSubheader>
                        WordPressとかAWSのブログ
                      </HeaderSubheader>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "function",
                      "props": Object {
                        "children": <Link
                          replace={false}
                          to="/"
                        >
                          WP-Kyoto
                        </Link>,
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "class",
                        "props": Object {
                          "children": "WP-Kyoto",
                          "replace": false,
                          "to": "/",
                        },
                        "ref": null,
                        "rendered": "WP-Kyoto",
                        "type": [Function],
                      },
                      "type": [Function],
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "function",
                      "props": Object {
                        "children": "WordPressとかAWSのブログ",
                      },
                      "ref": null,
                      "rendered": "WordPressとかAWSのブログ",
                      "type": [Function],
                    },
                  ],
                  "type": [Function],
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <GlobalNav />,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Container>
            <Grid
              columns={2}
              relaxed={true}
              verticalAlign="middle"
            >
              <GridRow
                stretched={true}
              >
                <GridColumn>
                  <Header
                    as="h1"
                  >
                    <HeaderContent>
                      <Link
                        replace={false}
                        to="/"
                      >
                        WP-Kyoto
                      </Link>
                    </HeaderContent>
                    <HeaderSubheader>
                      WordPressとかAWSのブログ
                    </HeaderSubheader>
                  </Header>
                </GridColumn>
                <GridColumn>
                  <GlobalNav />
                </GridColumn>
              </GridRow>
            </Grid>
          </Container>,
          <Divider />,
        ],
        "className": "App-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Grid
              columns={2}
              relaxed={true}
              verticalAlign="middle"
            >
              <GridRow
                stretched={true}
              >
                <GridColumn>
                  <Header
                    as="h1"
                  >
                    <HeaderContent>
                      <Link
                        replace={false}
                        to="/"
                      >
                        WP-Kyoto
                      </Link>
                    </HeaderContent>
                    <HeaderSubheader>
                      WordPressとかAWSのブログ
                    </HeaderSubheader>
                  </Header>
                </GridColumn>
                <GridColumn>
                  <GlobalNav />
                </GridColumn>
              </GridRow>
            </Grid>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <GridRow
                stretched={true}
              >
                <GridColumn>
                  <Header
                    as="h1"
                  >
                    <HeaderContent>
                      <Link
                        replace={false}
                        to="/"
                      >
                        WP-Kyoto
                      </Link>
                    </HeaderContent>
                    <HeaderSubheader>
                      WordPressとかAWSのブログ
                    </HeaderSubheader>
                  </Header>
                </GridColumn>
                <GridColumn>
                  <GlobalNav />
                </GridColumn>
              </GridRow>,
              "columns": 2,
              "relaxed": true,
              "verticalAlign": "middle",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <GridColumn>
                    <Header
                      as="h1"
                    >
                      <HeaderContent>
                        <Link
                          replace={false}
                          to="/"
                        >
                          WP-Kyoto
                        </Link>
                      </HeaderContent>
                      <HeaderSubheader>
                        WordPressとかAWSのブログ
                      </HeaderSubheader>
                    </Header>
                  </GridColumn>,
                  <GridColumn>
                    <GlobalNav />
                  </GridColumn>,
                ],
                "stretched": true,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": <Header
                      as="h1"
                    >
                      <HeaderContent>
                        <Link
                          replace={false}
                          to="/"
                        >
                          WP-Kyoto
                        </Link>
                      </HeaderContent>
                      <HeaderSubheader>
                        WordPressとかAWSのブログ
                      </HeaderSubheader>
                    </Header>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "as": "h1",
                      "children": Array [
                        <HeaderContent>
                          <Link
                            replace={false}
                            to="/"
                          >
                            WP-Kyoto
                          </Link>
                        </HeaderContent>,
                        <HeaderSubheader>
                          WordPressとかAWSのブログ
                        </HeaderSubheader>,
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "function",
                        "props": Object {
                          "children": <Link
                            replace={false}
                            to="/"
                          >
                            WP-Kyoto
                          </Link>,
                        },
                        "ref": null,
                        "rendered": Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "class",
                          "props": Object {
                            "children": "WP-Kyoto",
                            "replace": false,
                            "to": "/",
                          },
                          "ref": null,
                          "rendered": "WP-Kyoto",
                          "type": [Function],
                        },
                        "type": [Function],
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "function",
                        "props": Object {
                          "children": "WordPressとかAWSのブログ",
                        },
                        "ref": null,
                        "rendered": "WordPressとかAWSのブログ",
                        "type": [Function],
                      },
                    ],
                    "type": [Function],
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": <GlobalNav />,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
